include/libtorrent/torrent_handle.hpp

```diff
@@ -873,9 +871,12 @@ namespace libtorrent
 			, std::string const& private_key
 			, std::string const& dh_params);
 
+#ifndef TORRENT_NO_DEPRECATE
 		// Returns the storage implementation for this torrent. This depends on the
 		// storage constructor function that was passed to add_torrent.
+		TORRENT_DEPRECATED
 		storage_interface* get_storage_impl() const;
```

Comment:
Imagine a "RAM-disk" storage object. It might be very convenient to be able to access the stored blocks or populate it from the outside, rather than having to read or write them through libtorrent.